<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProjectName</class>
 <widget class="QWidget" name="DialogProjectName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projekt Name Ã¤ndern</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Geben Sie eine Projektbeschreibung an.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditProjektName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Abbrechen">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
